composer/algorithms/cutmix/cutmix.py

```diff
@@ -184,6 +184,12 @@ class CutMix(Algorithm):
             box such that each pixel has an equal probability of being mixed.
             If ``False``, defaults to the sampling used in the original
             paper implementation. Default: ``False``.
+        input_key (Any, Callable, Sequence[Callable]): A key that indexes to the input 
```

Comment:
Could you elaborate how these arguments are used, and perhaps give recommendations for how to set these parameters? Perhaps we can also explain that the default values work for where the batch is an (input, target) pair.

Reply:
I find the `Callable` typing confusing. That is an edge case, and makes it hard to read. We probably want a central doc page somewhere that these docstrings can refer to for more detail, as this will be shared across lots of algorithm classes.

Reply:
ok I can add something to `docs/source/trainer/algorithms.rst` once we change all the other Algorithms